Tests/RockLib.Configuration.AspNetCore.Tests/AspNetExtensionsTests.cs

```diff
@@ -33,15 +33,15 @@ public TestWebHostBuilder(IConfiguration configRoot)
             public IWebHostBuilder ConfigureServices(Action<WebHostBuilderContext, IServiceCollection> configureServices)
             {
                 var context = new WebHostBuilderContext { Configuration = _configRoot };
-                configureServices(context, null);
+                configureServices(context, null!);
```

Comment:
If we can pass in `null`, we should consider having the 2nd parameter to the action being nullable - i.e. `IServiceCollection?`